### PR TITLE
Update to include response object handling.

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

File: docs/user-guide/builtins.md [hunk 1]
```diff
@@ -133,23 +133,32 @@ should also be defined as `async`.
...
+Alternatively, rather than using a double `await` (one to get the response, the
+other to grab the data), it's possible to chain the calls into a single
+`await` like this:
 
-Assuming an `OK` response, the following methods are available to you to access
-the data returned from the server:
+```python title="A simple HTTP GET as a single `await`"
+from pyscript import fetch
+
+data = await fetch("https://example.com").text()
```
Comment (WebReflection) [inline docs/user-guide/builtins.md:156]: something odd with the highlighter ... but also something odd with the initial `python` thing as title is probably not understood in here ... if that's expected, feel free to ignore this comment!

Comment (WebReflection) [reply docs/user-guide/builtins.md:156]: to clarify, that's what GitHub produces and I think if we have a repo with `.md` files, those files should be compliant with what GitHub UI offers ... was that just a typo in the file? 🤔

Comment (ntoll) [reply docs/user-guide/builtins.md:156]: Agreed. Changing now.

**Review by WebReflection — APPROVED** [review]
the `!!! Danger` section still looks weird on GitHub but I guess we have our own MD parsing for docs so .... approved!

**Pull request merged**